dash/components/src/Computers/EditComputer.tsx

```diff
@@ -146,25 +146,28 @@ export default EditComputer;
 interface ComputerUserProps {
   name: string;
   id: string;
-  isOnline: boolean;
+  status: ChildComputerStatus;
 }
 
-const ComputerUser: React.FC<ComputerUserProps> = ({ name, id, isOnline }) => (
-  <Link
-    to={`/children/${id}`}
-    className="flex justify-between items-center odd:bg-slate-50 hover:bg-slate-100 transition-[background-color] duration-100 px-4 py-3 rounded-xl"
-  >
-    <h4 className="text-slate-700 font-semibold sm:text-lg">{name}</h4>
-    <div className="flex items-center gap-2">
-      <span className={cx(`text-sm`, isOnline ? `text-slate-600` : `text-slate-400`)}>
-        {isOnline ? `online` : `offline`}
-      </span>
-      <div
-        className={cx(
-          `w-3 h-3 rounded-full`,
-          isOnline ? `bg-green-400` : ` shadow-inner bg-slate-100`,
-        )}
-      />
-    </div>
-  </Link>
-);
+const ComputerUser: React.FC<ComputerUserProps> = ({ name, id, status }) => {
+  const isOnline = status.case !== `offline`;
```

Comment:
we're getting much richer detail from the API now, but to leverage it well will take some UI design work, to show stuff like "Suspended, resuming in 30 minutes" in the various places where this data is surfaced. So, for now, what I've done is basically downcast the data back to a boolean of "offline/online" like we used to get. This preserves the current behavior, and allows us to release the Macapp Version 2.7.0 which will start sending the richer data, and then we can as soon as we're ready start surfacing it to the parents in the dashboard. Basically, whenever you're ready to do a little design/component work on it.